### PR TITLE
Update icacherefresher

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
## Description

Update icacherefresher:
* [x] Write shorter sentences 
* [x] remove 'quite'

## Type of suggestion

* [x] Styleguide

## Product & version (if relevant)

Umbraco CMS v10,12 & 13

### CODE REVIEW

File: 10/umbraco-cms/reference/cache/icacherefresher.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ _This section describes what ICacheRefresher and ICacheRefresher&lt;T&gt; are an
 
 ## What is an ICacheRefresher
 
-This interface has been in the Umbraco core for quite some time but has really only been used to ensure that content cache is refreshed among all server nodes participating in a load balanced scenario.
+This interface has been in the Umbraco core for a significant period. However, it has really only been used to ensure that content cache is refreshed among all server nodes participating in a load balanced scenario.
 
-With changes in 6.1, this has changed slightly, an `ICacheRefresher` is now the primary way to invalidate *any* cache that needs to be refreshed or removed regardless of whether we are in a load balanced environment.
+With changes in 6.1, this has changed slightly. An `ICacheRefresher` is now the primary method. It invalidates *any* cache needing refreshment or removal. This applies regardless of a load balanced environment.
```
Comment (alina-tincas) [inline 10/umbraco-cms/reference/cache/icacherefresher.md:14]: Seems this content has been here for a while as it is mentioning v6.1. In this case we should remove the version and only keep what is essential in this case.
```suggestion
An `ICacheRefresher` is the primary method that invalidates *any* cache needing refreshment or removal. This applies regardless of a load balanced environment.
```

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @erikjanwestendorp thank you for the PR 🙌 

I have one suggestions, can you please take a look into it and if you agree make the same change to the other versions as well? 😊

**Comment by erikjanwestendorp** [comment]
@alina-tincas Thanks for your review and suggestion, just updated the PR 😄

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
Thank you for making those changes 🙌 

Merging this in #h5YR

**Pull request merged**